LethalAPI.TerminalCommands/TerminalNodeExtensions.cs

```diff
@@ -0,0 +1,189 @@
+﻿using LethalAPI.TerminalCommands.Models.Enums;
+using UnityEngine;
+using UnityEngine.Video;
+
+namespace LethalAPI.TerminalCommands
+{
+	/// <summary>
+	/// Provides a collection of Builder extension methods for <seealso cref="TerminalNode"/>
+	/// </summary>
+	public static class TerminalNodeExtensions
+	{
+		/// <summary>
+		/// Sets the display/response text of a terminal node
+		/// </summary>
+		/// <param name="node">Terminal node to modify</param>
+		/// <param name="displayText">New display text</param>
+		/// <returns>Original terminal node</returns>
+		public static TerminalNode WithDisplayText(this TerminalNode node, object displayText)
+		{
+			node.displayText = displayText.ToString();
+			return node;
+		}
+
+		/// <summary>
+		/// Sets the display texture, visible in the background of the terminal
+		/// </summary>
+		/// <param name="node">Terminal node to modify</param>.P
+		/// <param name="texture">Texture to make visible in the background of the terminal</param>
+		/// <returns>Original terminal node</returns>
+		public static TerminalNode WithDisplayTexture(this TerminalNode node, UnityEngine.Texture texture)
+		{
+			node.displayTexture = texture;
+			return node;
+		}
+
+		/// <summary>
+		/// Displays a <see cref="VideoClip"/> in the background of the terminal. This is the same way the 'View Monitor' command works
+		/// </summary>
+		/// <param name="node">Terminal node to modify</param>
+		/// <param name="video">Video clip to display in the background</param>
+		/// <returns>Original terminal node</returns>
+		public static TerminalNode WithVideoClip(this TerminalNode node, VideoClip video)
+		{
+			node.displayVideo = video;
+			return node;
+		}
+
+		/// <summary>
+		/// Enables accept anything
```

Comment:
Could you elaborate on the effect this has?

Reply:
I will likely just remove this entry, as it pertains to the vanilla confirm/deny system, which is going to be redundant for modders in place of the custom Terminal Interactions system.

Reply:
Having builders for manual node / key creation is still good imo. I'd keep it in.

I know some mods I have planned likely still need to edit / work with vanilla terminal nodes, for example.

Reply:
I've done a bit more research into the purpose of this field, and re-done the docs